app/uk/gov/hmrc/checkeorinumberfrontend/config/AppConfig.scala

```diff
@@ -50,7 +50,8 @@ class AppConfig @Inject()(config: Configuration, servicesConfig: ServicesConfig)
   def routeToSwitchLanguage: String => Call = (lang: String) => routes.CustomLanguageSwitchController.switchToLanguage(lang)
 
   lazy val languageTranslationEnabled: Boolean =
-    config.getBoolean("microservice.services.features.welsh-translation").getOrElse(true)
+    config.get[Boolean]("microservice.services.features.welsh-translation")
```

Comment:
Use config.getOptional[Boolean] so you can keep get orElse default value as true

Reply:
Just changed as suggested, sorry for delay.